DXMainClient/Program.cs

```diff
@@ -117,101 +111,87 @@ static void Main(string[] args)
             // We're a single instance application!
             // http://stackoverflow.com/questions/229565/what-is-a-good-pattern-for-using-a-global-mutex-in-c/229567
 
-            string appGuid = ((GuidAttribute)Assembly.GetExecutingAssembly().GetCustomAttributes(
-                typeof(GuidAttribute), false).GetValue(0)).Value.ToString();
-
             // Global prefix means that the mutex is global to the machine
-            string mutexId = string.Format("Global/{{{0}}}", appGuid);
-
-
-            var allowEveryoneRule = new MutexAccessRule(new SecurityIdentifier(WellKnownSidType.WorldSid, null),
-                MutexRights.FullControl, AccessControlType.Allow);
-            var securitySettings = new MutexSecurity();
+            string mutexId = string.Format("Global{0}", Guid.Parse("1CC9F8E7-9F69-4BBC-B045-E734204027A9"));
+
+#if NETFRAMEWORK
+            var allowEveryoneRule = new System.Security.AccessControl.MutexAccessRule(
+                new System.Security.Principal.SecurityIdentifier(System.Security.Principal.WellKnownSidType.WorldSid, null),
+                System.Security.AccessControl.MutexRights.FullControl,
+                System.Security.AccessControl.AccessControlType.Allow);
+            var securitySettings = new System.Security.AccessControl.MutexSecurity();
             securitySettings.AddAccessRule(allowEveryoneRule);
```

Comment:
Not familiar with this, what does this do? And the subsequent code too. May be good to leave a comment here.

Reply:
This code was already in place. It is Windows only stuff and not supported by the Mutex class in .NET6.